linera-chain/src/data_types.rs

```diff
@@ -439,6 +441,16 @@ pub struct LiteValue {
     pub kind: CertificateKind,
 }
 
+impl LiteValue {
+    pub fn new<T: CertificateValue>(value: &Hashed<T>) -> Self {
```

Comment:
This contructor is a bit weird. What was the issue with `.lite()`? Would a conversion `From` make sense?

Reply:
Personally I don't like `From` conversions which are not obvious. It makes it difficult to track the flow - it's impossible to look for w/ IDE's "find usage/references" feature.